src/shared/components/Table/Table.tsx

```diff
@@ -89,8 +88,8 @@ export const Table = ({
               {rows
                 ?.sort(getComparator(order, orderBy))
                 ?.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
-                .map((row) => (
-                  <TableRow key={uniqueId('row_')} data-testid="table-row">
+                .map((row, index) => (
+                  <TableRow key={`row-${index}`} data-testid="table-row">
```

Comment:
Are you sure that index better than uniqueId?

Reply:
![image](https://user-images.githubusercontent.com/116631639/224959617-47b92c2e-d722-4881-9ce5-9dc894cf333a.png)